final_prep/README.md

```diff
@@ -38,18 +38,18 @@ When you are finished, add screenshots of your calendar so we can provide feedba
 
 ### Mentorship Prep
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
-- [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+- [x] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
+  - [x] Add link to your gist here: https://gist.github.com/michaeljhicks/11d8d97b97c9746cf23af311992d9c15
```

Comment:
I'm from ABQ, too! Well, was born there, grew up in Las Cruces more.

Reply:
No way! I love seeing other Abq/NM people in Denver!